libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/internal/MapboxVoiceInstructions.kt

```diff
@@ -19,14 +20,26 @@ import kotlinx.coroutines.flow.mapLatest
 @OptIn(ExperimentalCoroutinesApi::class)
 class MapboxVoiceInstructions {
 
+    private val reasonsWithoutNewInstructions = setOf(
+        RoutesExtra.ROUTES_UPDATE_REASON_CLEAN_UP,
+        RoutesExtra.ROUTES_UPDATE_REASON_REFRESH,
+        RoutesExtra.ROUTES_UPDATE_REASON_ALTERNATIVE,
```

Comment:
Does alternative always has an initial banner instruction? What if we drive for a while and then switch to alternative?

Reply:
Then the reason will be new and we start predownloading for this new primary route.
When the reason is alternative, it doesn't affect our predownloading at all: we only predownload for the current primary route.
We can discuss predownloading for alternatives as well, but: a) it's out of scope of this bug-fix, b) I'm not sure if it will make a lot of sense compared to the disadvantage of consuming more traffic.